node/middlewares/scheduler/createScheduler.ts

```diff
@@ -0,0 +1,37 @@
+import { SchedulerRequestMethods } from '../../typings/scheduler'
+import { CRON } from '../../utils/constants'
+import { formatError } from '../../utils/formatError'
+
+export async function createScheduler(ctx: Context, next: () => Promise<any>) {
+    const {
+      clients: { scheduler },
+      vtex: { logger },
+    } = ctx
+  
+    try {
+      await scheduler.createOrUpdateScheduler({
+        cronId: CRON.id,
+        cronExpression: CRON.expression,
+        request: {
+          uri: CRON.url(ctx.host),
+          method: SchedulerRequestMethods.POST,
+          body: {
+            token: CRON.authToken,
+            }
+        },
+      })
+  
+      ctx.status = 201
+      ctx.body = 'Cron created'
+  
+      await next()
+    } catch (e) {
+      logger.error({
+        middleware: 'CREATE SELLER ACCOUNT NAME CRON',
```

Comment:
Can you please change the string of middleware from 'CREATE SELLER ACCOUNT NAME CRON', into something related with this project, like 'CREATE RETURN APP SCHEDULER CRON'.